### PR TITLE
Initial Build Lenovo ZUK Z2 Plus

### PR DESCRIPTION
**Pull request by SSpidey**
This is for crDroid First Build in Q for Official Maintainer ship.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-cayman
```
Comment (gwolf2u) [inline _config.yml]: whats this for?
please remove

File: z2_plus.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+			"oem": "Lenovo",
+			"device": "Lenovo ZUK Z2 Plus",
+			"filename": "crDroidAndroid-10.0-20200822-z2_plus-v6.9.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/z2_plus/6.x/crDroidAndroid-10.0-20200822-z2_plus-v6.9.zip/download",
+			"timestamp": 1598072328,
+			"md5": "364ed0fbfc380553651d13ecc6d6118f",
+			"size": 711364517,
+			"version": "6.9",
+			"buildtype": "Monthly",
+			"forum": "https://forum.xda-developers.com/member.php?u=8912149",
```
Comment (gwolf2u) [inline z2_plus.json]: this should be forum url not your username url

**Review by gwolf2u — CHANGES_REQUESTED** [review]
please correct needed changes

**Comment by SSpidey** [comment]
Changes done

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**